include/boost/geometry/util/precise_math.hpp

```diff
@@ -347,29 +355,51 @@ inline RealNumber orient2dtail(vec2d<RealNumber> const& p1,
     return(D[D_nz - 1]);
 }
 
-// see page 38, Figure 21 for the calculations, notation follows the notation in the figure.
+// see page 38, Figure 21 for the calculations, notation follows the notation
+// in the figure.
 template
 <
     typename RealNumber,
-    std::size_t Robustness = 3
+    std::size_t Robustness = 3,
+    typename EpsPolicy
 >
 inline RealNumber orient2d(vec2d<RealNumber> const& p1,
                            vec2d<RealNumber> const& p2,
-                           vec2d<RealNumber> const& p3)
+                           vec2d<RealNumber> const& p3,
+                           EpsPolicy& eps_policy)
```

Comment:
Do I understand correctly that `eps_policy` is output argument. And it's a policy defined internally in `side_robust`? Shouldn't this policy be defined somewhere in this file instead?

Reply:
It is created in `side_robust` and passed to `orient2d` that is why is defined there.